.github/workflows/linux.yml

```diff
@@ -74,7 +67,7 @@ jobs:
         run: |
           sudo apt-get install -y \
             ${{ matrix.conf.packages }} \
-            $(./scripts/list-build-dependencies.sh -m apt ${{ matrix.conf.flags }})
+            $(./.github/scripts/packages-ubuntu-apt.sh)
 
```

Comment:
We could even go simpler with with a text file; less syntax in the list and the invocation. 

`sudo xargs apt-get install -y < /.github/package-lists/ubuntu.txt`

ubuntu.txt
``` text 
build-essential
ccache
libasound2-dev
libfluidsynth-dev
libopusfile-dev
libpng-dev
libsdl2-dev
libsdl2-net-dev
```

Easy for average users to deal with too, when we promote meson and use this to help general users install deps.

Reply:
I made it a script so I could put comments in this file, but in retrospect, I agree - a simple, flat file will be better :)

Reply:
Worth noting: even with warning_level=3 - `-pedantic` issues are reported, but they are not gated via count-warnings script (the warning pattern does not match the regex).

I think -pedantic is useless to us either way - I suspect it's designed for situations when ISO-compliance is required by law..

Reply:
That makes sense if they're useless, going by the definition of a pedant: one who is overly concerned with literal accuracy or formality.
GCCs authors have done a good job at binning those warnings into the pedant bucket and keeping them out of `-Wall` :-)

Reply:
I renamed the files to: `.github/packages/ubuntu-apt.txt` and `.github/packages/macos-brew.txt`.

Reply:
Perfect.. nice and flexible to handle more permutations!